ui/core-react/src/test/radialmenu/RadialMenu.test.tsx

```diff
@@ -130,67 +115,65 @@ describe("RadialMenu", () => {
             </RadialButton>
           );
         })}
-      </RadialMenu>;
-    });
-
-    it("should render", () => {
-      const wrapper = mount(radialMenu2);
-      wrapper.unmount();
-    });
-
-    it("renders correctly", () => {
-      shallow(radialMenu2).should.matchSnapshot();
+      </RadialMenu>);
+
+      const expectedSize = 16.838071024063936;
+      data.map(({label, transform, x, y})=> {
+        const button = screen.getByText(label, {selector: ".core-radial-menu-button-svg>.core-radial-menu-button-container>.core-radial-menu-button-content"});
+        const svg: HTMLElement | undefined | null = button.parentElement?.parentElement;
+
+        expect(svg).to.exist;
+        expect(svg?.getAttribute("transform")).to.eq(transform);
+        expect(Number.parseFloat(svg?.getAttribute("x")??"")).to.be.closeTo(x, 0.05);
+        expect(Number.parseFloat(svg?.getAttribute("y")??"")).to.be.closeTo(y, 0.05);
+        expect(Number.parseFloat(svg?.getAttribute("width")??"")).to.be.closeTo(expectedSize, 0.05);
+        expect(Number.parseFloat(svg?.getAttribute("height")??"")).to.be.closeTo(expectedSize, 0.05);
+      });
     });
 
-    it("should call onSelect", () => {
+    it("should call onSelect", async () => {
       const spyMethod = sinon.fake();
-      const wrapper = mount(<RadialMenu opened={true} innerRadius={10} outerRadius={100}>
+      render(<RadialMenu opened={true} innerRadius={10} outerRadius={100}>
         <RadialButton key="0" icon="icon-placeholder" onSelect={spyMethod}> Test </RadialButton>
       </RadialMenu >);
-      const button = wrapper.find(RadialButton);
-      const gEl = button.find("g");
-      gEl.simulate("click");
+
+      await theUserTo.click(screen.getByText("Test"));
       spyMethod.should.have.been.called;
-      wrapper.unmount();
     });
 
     it("should call onSelect when button select API called", () => {
       const spyMethod = sinon.fake();
-      const wrapper = mount(<RadialMenu opened={true} innerRadius={10} outerRadius={100}>
-        <RadialButton key="0" icon="icon-placeholder" onSelect={spyMethod}> Test </RadialButton>
-      </RadialMenu >);
+      const button = React.createRef<RadialButton>();
+      render(<svg><RadialButton ref={button} key="0" icon="icon-placeholder" onSelect={spyMethod}> Test </RadialButton></svg>);
 
-      const button = wrapper.find(RadialButton);
-      (button.instance() as RadialButton).select();
+      button.current?.select();
       spyMethod.should.have.been.called;
-
-      wrapper.unmount();
     });
 
     it("should call onSelect when menu select API called", () => {
       const spyMethod = sinon.fake();
-      const wrapper = mount(<RadialMenu opened={true} innerRadius={10} outerRadius={100} selected={0}>
+
+      const menu = React.createRef<RadialMenu>();
+      render(<RadialMenu ref={menu} opened={true} innerRadius={10} outerRadius={100} selected={0}>
```

Comment:
This only works for class component, but gives a "user centric" way of doing the same test, rather than using enzyme instance.